code-of-conduct.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+---
+
+For any issues/questions/queries, please contact an organiser:
+
+|             | Slack | Twitter                                                |
+| ----------- | ----- | ------------------------------------------------------ |
+| Ben Foxall  | @ben  | [@benjaminbenben](https://twitter.com/@benjaminbenben) |
+| Ryan Brooks | @ryan | [@spikeheap](https://twitter.com/@spikeheap)           |
+
+# Code of Conduct
+
+## Purpose
+
+A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+
+This Code of Conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in our events to help us create safe and positive experiences for everyone.
+
+## Open [Source/Culture/Tech] Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+## Expected Behavior
```

Comment:
ah HA! A secret American! Can we "u" this up, please?

Reply:
"u" got it :+1: